dimod/bqm/adjvectorbqm.pxd

```diff
@@ -27,15 +27,63 @@ ctypedef vector[pair[VarIndex, Bias]].const_iterator NeighborIterator
 
 
 cdef class cyAdjVectorBQM:
+    """A binary quadratic model where the neighborhoods are c++ vectors.
```

Comment:
"A binary quadratic model implemented with c++ vectors for variable neighborhoods."?

Reply:
I'd like to keep it under 80 character width for the first line of the docstring

Reply:
""A binary quadratic model implemented with c++ vectors" and then a second line with the details?